python/torch_mlir_e2e_test/test_suite/__init__.py

```diff
@@ -14,6 +14,47 @@
     "NativeGroupNormBackwardModule_basic",
     "QuantizedMLP_basic",
     "ReduceMaxAlongDimUnsignedInt_basic",
+    "IndexPut1DFloatAccumulateModule_basic",
```

Comment:
This is a big step back. The linalg lowering should be fixed before landing the patch.

Reply:
I didn't find the lower for indexput in linalg, Could you please point out where is it? So that I can replace it with indexput.hacked_twin

Reply:
It's here: https://github.com/llvm/torch-mlir/blob/4dac1a551ab101fecb280315434f0b398296c75a/lib/Conversion/TorchToTMTensor/TorchToTMTensor.cpp#L698

Reply:
Why is it inside the conversion TMTensor? Shouldn't it be located in the conversion to linalg?

Reply:
`TMTensor` is part of the `linalg-on-tensors` output type in Torch-MLIR:

https://github.com/llvm/torch-mlir/blob/a8fd275a006b4c3284991bf7ac3608d4f8ecf8cf/lib/Dialect/TorchConversion/Transforms/Passes.cpp#L65-L72

`TMTensor` dialect captures ops that are not expressible in the `linalg` dialect.

Reply:
Get most of the regression fixed except for 
```
    "IndexPutImplIndexWithNoneModule_basic",
    "SliceCopyNonZeroDim_Module_basic",
```
Could you help take a look at why this 2 failed.